app/src/main/java/com/nicoletavlad/newsreader/StartApplication.java

```diff
@@ -0,0 +1,24 @@
+package com.nicoletavlad.newsreader;
+
+import android.app.Application;
+
+import com.nicoletavlad.data.di.RepositoryModule;
+
+public class StartApplication extends Application
```

Comment:
Looks right, what it begs is code formatting, there are lots of random new lines and semicolon opened on next line is really uncommon for android projects. Let's just run Android Studio for this:
- In project tree view - right click on the root folder - select "Reformat Code". Pop up will appear select desired actions, select all, run it and commit. Respecting the code style for a project is really important when working in large teams (even makes pull requests cleaner) and always a good habit.

Reply:
I have used the Reformat Code option when I committed. I have also changed the Code Style in settings to be with more empty lines around imports, methods, and classes' names (they are not random), and brackets put on new lines. It's a lot easier for me to see this way, being more spaced, and not getting lost between the lines. I'm using this style for every personal project, no matter the language.
Should I change the code style with the default one for Android? Or another one?